app/frontend/src/components/forms/ShareForm.vue

```diff
@@ -8,59 +8,67 @@
       </template>
 
       <v-card>
-        <v-card-title class="headline pb-0">Get sharing link for this form</v-card-title>
+        <v-card-title class="headline pb-0"> Share Link </v-card-title>
```

Comment:
Potential unnecessary leading and trailing whitespaces?

Reply:
Fixed